src/osm_flex/clip.py

```diff
@@ -160,19 +160,23 @@ def _build_osmconvert_cmd(shape, osmpbf_clip_from, osmpbf_output):
         file path to planet.osm.pbf or other osm.pbf file to clip
     osmpbf_output : str or pathlib.Path
         file path (incl. name & ending) under which extract will be stored
+        
+    Note
+    -----
+    If cross-border multi-polygons or lines should be retained, comment out
+    below options '--complete-ways' and '--complete-multipolygons'. This only
+    works with parent files < 2GB.
```

Comment:
This only works for source installations, but not if the package is loaded from PyPI. Maybe add this as an optional parameter to the function? Something like
```python
def _build_osmconvert_cmd(shape, osmpbf_clip_from, osmpbf_output, complete=False):
    pass
    """
    Set ``complete=True`` to retain cross-border multi-polygons and lines.
    """
```

Reply:
I think adding something like `kernel_kwargs`, and documenting which options are available for each kernel, would be fine. But maybe this is an idea for the next release 🙃

Reply:
Ah yes, I didn't think of this possibility. Good point. Will make a note.